integrationtest/loadfiles/cases/checker/CheckedRequests.d

```diff
@@ -266,6 +266,8 @@ class UnexpectedNotification
 
 class Consume: RecordChecker
 {
+    import dmqproto.client.request.Consume;
```

Comment:
Are these changes necessary? The client was designed so that importing request modules in user code would not be necessary.

Reply:
Trick is that this doesn't work with the 2.071 import rules (I think).

Reply:
Can we fix this in the client? It would be nice for it to keep working and not require all users to change their code. The required change won't be at all obvious to the average user.

Reply:
I would hope so it's just the matter of making these imports `public`: https://github.com/sociomantic-tsunami/dmqproto/blob/v14.x.x/src/dmqproto/client/mixins/NeoSupport.d#L51

Reply:
I'll checkout this branch and give it a try, I agree this is a very bad thing to push to the users.

Reply:
OK, looks like that fixes it. I'll submit PR to dmqproto.

Without the change imports in `NeoMixin` to public:

```
integrationtest/loadfiles/cases/checker/CheckedRequests.d(352): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Consume is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(353): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Consume is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(385): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(422): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(423): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(502): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(503): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(352): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Consume is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(353): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Consume is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(385): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(422): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(423): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(502): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests
integrationtest/loadfiles/cases/checker/CheckedRequests.d(503): Deprecation: dmqproto.client.DmqClient.DmqClient.NeoSupport!().Neo.Pop is not visible from module CheckedRequests

```

With the change:

```
   rdmd build/devel/tmp/test-loadfiles
   run build/devel/tmp/test-loadfiles
```

Reply:
https://github.com/sociomantic-tsunami/dmqproto/pull/56

Reply:
These request module import changes should be removed now that dmqproto has been fixed.